python/cudf/cudf/core/tools/numeric.py

```diff
@@ -217,20 +216,28 @@ def _convert_str_col(col, errors, _downcast=None):
     else:
         if errors == "coerce":
             col = libcudf.string_casting.stod(col)
-            non_numerics = is_integer.binary_operator(
-                "or", is_float
-            ).unary_operator("not")
+            non_numerics = is_float.unary_operator("not")
             col[non_numerics] = None
             return col
         else:
             raise ValueError("Unable to convert some strings to numerics.")
 
 
-def _proc_inf_strings(col):
+def _proc_inf_null_strings(col):
     col = col.str().lower()
+    col = _proc_null_strings(col)
+    col = _proc_inf_strings(col)
+    return col
+
+
+def _proc_null_strings(col):
+    s = cudf.Series(col)
+    s = s.where(s != "", "NaN")
+    return s._column
```

Comment:
If the point of this function is to convert empty strings to `NaN` could we name it a bit more accordingly? From a glance I expected this to handle where the validity bit is unset but that's not the case.

Also a one liner docstring would help for these helper functions.